analyzers/tests/SonarAnalyzer.Net8.UnitTest/TestCases/TooManyParameters.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Runtime.CompilerServices;
+
+class ZeroOverheadMemberAccess
+{
+    [UnsafeAccessor(UnsafeAccessorKind.Constructor)]
+    extern static UserData CallConstructor(int x1, int x2, int x3, int x4);                 // Compliant: signature has to match target
+
+    [UnsafeAccessor(UnsafeAccessorKind.Method, Name = "Method")]
+    extern static void CallMethod(UserData userData, int x1, int x2, int x3, int x4);       // Compliant: signature has to match target
+
+    [UnsafeAccessor(UnsafeAccessorKind.StaticMethod, Name = "StaticMethod")]
+    extern static void CallStaticMethod(UserData userData, int x1, int x2, int x3, int x4); // Compliant: signature has to match target
+}
```

Comment:
There is no need to special case `UnsafeAccessor`. `extern` should be enough:
```suggestion
    [DllImport("user32.dll")]
    extern static void SomeMethod(UserData userData, int x1, int x2, int x3, int x4); // Compliant: signature has to match target
}
```